pom.xml

```diff
@@ -43,6 +43,7 @@
         <shadeBase>com.facebook.presto.hadoop.\$internal</shadeBase>
         <dep.slf4j.version>1.7.13</dep.slf4j.version>
         <dep.hadoop.version>2.7.3</dep.hadoop.version>
+        <dep.httpclient.version>4.5.2</dep.httpclient.version>
```

Comment:
No need for a variable, since the dependency is only used once. We use variables when there are multiple dependencies that need the same version.